src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/JJDisputeController.java

```diff
@@ -22,7 +22,21 @@ public class JJDisputeController {
 	private JJDisputeService jjDisputeService;
 
 	private Logger logger = LoggerFactory.getLogger(JJDisputeController.class);
-	
+
+	/**
+	 * GET endpoint that retrieves a jj dispute by id from the database
+	 * @param ticketNumber the primary key of the jj dispute to retrieve.
+	 * @return list of all jj disputes
```

Comment:
We should say in the comment that this returns a single jj dispute with specified id.

Reply:
Good catch. Copy n' paste :)